src/bin/release-please.ts

```diff
@@ -346,6 +346,10 @@ function pullRequestStrategyOptions(yargs: yargs.Argv): yargs.Argv {
       describe: 'Override the detected latest tag name',
       type: 'string',
     })
+    .option('date-format', {
+      describe: 'format in strftime format for updating dates',
+      default: 'string',
```

Comment:
Sorry, I missed this on the first review. This was probably meant to be `type` and/or the default value should be `%Y-%m-%d` (or left as nullable and rely on the default value set in code).